package.json

```diff
@@ -18,6 +18,6 @@
     "moment": "*"
   },
   "dependencies": {
-    "@types/lodash": "^4.14.144"
```

Comment:
Did you mean to remove lodash?

Reply:
Without lodash yarn build fails.

Reply:
Not sure what's going on there. I'm pretty sure it's not being used directly, but maybe the grafana dependencies use it. And it's not installing `lodash` itself, but only the Typescript type definitions. I've reverted the change.